solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -434,21 +435,28 @@ contract RandomBeacon is Ownable {
         );
     }
 
-    /// @notice Notifies about DKG timeout.
+    /// @notice Notifies about DKG timeout. Pays the sortition pool unlocking
+    ///         reward to the notifier.
     function notifyDkgTimeout() external {
         dkg.notifyTimeout();
-
-        // TODO: Pay a reward to the caller.
+        // Pay the sortition pool unlocking reward.
+        relay.tToken.safeTransfer(msg.sender, sortitionPoolUnlockingReward);
+        dkg.complete();
     }
 
     /// @notice Approves DKG result. Can be called after challenge period for the
     ///         submitted result is finished. Considers the submitted result as
-    ///         valid and completes the group creation by activating the candidate
-    ///         group.
+    ///         valid, pays reward to the result submitter and completes the
+    ///         group creation by activating the candidate group.
     function approveDkgResult() external {
         dkg.approveResult();
-
+        // Pay the DKG result submission reward.
+        relay.tToken.safeTransfer(
```

Comment:
And same here.

Reply:
Done in https://github.com/keep-network/keep-core/pull/2690/commits/443a64b2e12a7577f113498f5c40e0bcab5a19de.